usaspending_api/awards/tests/data/award_test_data.py

```diff
@@ -17,150 +18,190 @@
 }
 OBLIGATED_AMOUNT_BY_AWARD_ID = {0: 0.0, 1: 110011.0, 2: 430043.0, 3: 960096.0, 4: 1700170.0}
 
+DATE_IN_THE_PAST = "1776-07-04"
+DATE_IN_THE_FUTURE = "2553-04-01"
```

Comment:
Oddly the French Embassy in the US has nothing scheduled for that date https://franceintheus.org/spip.php?page=MF-jour&date=2553-04-01

Reply:
Good catch. I will notify the french at once.